apps/admin-ui/src/component/reservations/ReservationsDataLoader.tsx

```diff
@@ -129,7 +129,7 @@ const ReservationsDataLoader = ({
     sort
   );
 
-  if (loading && !data) {
+  if (loading && data.length === 0) {
```

Comment:
is `data` ever `undefined` or `null`?
(Do we need to check for both here: `(!data || data.length === 0)` or `data?.length === 0`?)

Reply:
It's never null (goes through a filter function that returns [] for null). 
Typescript would raise an error for `data.length === 0` and also `data?.length === 0`